package.json

```diff
@@ -64,18 +64,18 @@
 		"i18n:gutenberg": "./bin/i18n-download.sh gutenberg",
 		"i18n:gutenberg:force": "cross-env REFRESH_I18N=1 ./bin/i18n-download.sh gutenberg",
 		"bundle": "npm run clean; npm run bundle:js && npm run genstrings",
-		"prebundle:js": "npm run i18n:gutenberg",
+		"prebundle:js": "npm run i18n:gutenberg:force",
```

Comment:
As part of the bundling process, we should re-download and update the i18n cache files to provide the most recent translations. 

This behavior is identical to the JS bundle generation in Gutenberg ([reference](https://github.com/WordPress/gutenberg/blob/e393d55cacc030f43b7f64104a8ff1aa2b179a62/packages/react-native-editor/package.json#L87)). We haven't updated the cache since a long time ago because the translations from the GlotPress project was missing some of the strings (more info [here](https://github.com/WordPress/gutenberg/issues/33364)), now we can re-enable it as the issue was fixed on previous versions.